mta/fixtures/business_service.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from conftest import generate_description, generate_name
+from swagger_client.models.api_business_service import ApiBusinessService
+
+
+@pytest.fixture()
+def business_service(stakeholder, create_api, get_api):
+    api_business_service = ApiBusinessService(
+        name=generate_name(start="business service"), description=generate_description(), owner=stakeholder
+    )
+    new_business = create_api.businessservices_post(api_business_service.to_dict())
```

Comment:
Please rename to `new_business_service` or `new_bs`

Reply:
Done. Thanks!